sealtk/core/KwiverPipelineWorker.cpp

```diff
@@ -303,6 +303,7 @@ void KwiverPipelineWorker::sendInput(kwiver::embedded_pipeline& pipeline)
   auto sourcesToUse = QVector<VideoSource*>{};
   auto requestorsInUse = QVector<PipelineVideoRequestor*>{};
   auto lastTime = std::numeric_limits<ts_time_t>::min();
+  auto framesProcessed = int{0};
```

Comment:
Heh... this takes AAA to an almost fanatical level :)

Reply:
Actually, one of the arguments for AAA is that you can't forget to initialize something. IOW, I can't write `int x; // oops, forgot the initializer`. There is also a consistency argument to always having the type on the RHS (and `auto` on the LHS); unfortunately, that doesn't work with immovable types... like `QObject`, which means Qt code frequently *can't* use AAA.

In this case, it was mainly for consistency since the other declarations are all `auto`... although I do favor AAA in general. Nevertheless, you may note that I used `int ... = ...` elsewhere, where it wasn't part of a block of declarations already using `auto`.